app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::GreetingsController < ApplicationController
+  def random_greeting
+    @random_greeting = Greeting.order('RANDOM()').first.message
```

Comment:
## [OPTIONAL]
- [ ] Nice work getting the random message, But I would sugest you to return the woll object not only message string
```suggestion
    @random_greeting = Greeting.order('RANDOM()').first
```

Reply:
Thanks for that suggestion @andersonlebon. I have updated it, cheers. 🥂